### ISSUE TITLE
运行test.py时，anno_type参数无法获取，疑似是dataloader函数加载不正确使得eval_SHT函数中score值一直为空

### ISSUE THREAD

**Issue by chiao666** [fjchange/MIST_VAD#21]
在此处（test.py117行）代码添加print的话，发现anno_type参数为空，使得normal_sxores一直为空。
`def eval_SHT(model, test_dataloader):
    total_labels, total_scores, normal_scores = [], [], []
    for frames, anno_type, _, annos in test_dataloader:
        # print('anno_type'.format(anno_type))
        frames = frames.float().contiguous().view([-1, 3, frames.shape[-3], frames.shape[-2], frames.shape[-1]]).cuda()
        with torch.no_grad():
            scores, feat_maps = model(frames)[:2]
        if args.ten_crop:
            scores = scores.view([-1, 10, 2]).mean(dim=-2)
        for clip, score, anno in zip(frames, scores, annos):
            score = [score.squeeze()[1].detach().cpu().item()] * args.segment_len
            total_scores.extend(score)
            total_labels.extend(anno.tolist())
            if anno_type == 'Normal':
                normal_scores.extend(score)
    return eval(total_scores, total_labels, normal_scores)`

**Comment by fjchange** [comment]
The codes have been updated. This bug should be solved.

**Comment by Belize9709** [comment]
请教一下，是否有人从stage1一直到test。用这个模型跑出来数据，为什么我跑出来的SHT数据最终auc只有50%不到。
